spec/stored_queries/taxonomy/taxonomy_search_species_strain.yaml

```diff
@@ -53,7 +53,8 @@ query: |
   )
   FOR doc IN FULLTEXT(@@taxon_coll, @sciname_field, search_text__fulltext)
       FILTER @ts ? doc.created <= @ts AND doc.expired >= @ts : true
-      FILTER doc.rank IN ["species", "strain"] OR doc.strain
+      // note that doc.strain is deprecated TODO add link
```

Comment:
TODO before merge: add link to file in https://github.com/kbase/relation_engine_importers/pull/52 when that's merged

Reply:
Curious why is this a code comment? Seems odd to document in code the history of changes. Should it go just into CHANGELOG?

Reply:
Because the next person to look at this is going to wonder why the heck there's two seemingly at least semi-redundant flags in the query and it going to want to remove one. Providing a link to the explanation of how the two flags came to be could save them at best a good chunk of time trying to figure out what's going on and at worst making a mistake that breaks the query.

Done, added link

Reply:
Oh ... I was just thinking put in the file comments at the top that this query is written de facto to interact with the NCBI imported data and that's what explains the rank filtering criteria. We've never even tried this query on other taxonomy imported data, right?

Putting so much documentation in the code itself seems claustrophobic

Reply:
> We've never even tried this query on other taxonomy imported data, right?

Not as far as I know

I feel pretty strongly that the documentation/warning for the code should live as close to the code as possible otherwise it's too easy to miss. This is the kind of stuff that code comments are for, IMO

Reply:
Yeah I agree actually